components/Overlay/index.css

```diff
@@ -0,0 +1,17 @@
+.overlay {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  background-color: var(--grey-90-a60);
+}
+
+.popup {
```

Comment:
I'd move this into the story - could be a separate `stories.css` only imported in `stories.js`. I don't think we want to include this sample content in the distributed build.